crowdsec-docs/docs/bouncers/nginx.mdx

```diff
@@ -79,102 +94,314 @@ If you are on a mono-machine setup, the `crowdsec-nginx-bouncer` install script
+sudo cscli decisions add -i <IP_TO_TEST> -t captcha
+```
+
+## FAQ
+
+### Why remediation are not apply instantaneous
+
+In stream mode, the bouncer will lunch the internal timer to pull the local API after the first request. After installing the bouncer, just do a simple curl on your website to trigger the puller timer.
+
+### Resolver and certificates
+
+In order to query `google.com` for the reCAPTCHA verification, we need to set a resolver and a SSL certificate in our nginx configuration.
+Those are the default values, but you can change them:
+
+```bash title="etc/nginx/conf.d/crowdsec_nginx.conf"
+resolver 8.8.8.8 ipv6=off;
+lua_ssl_trusted_certificate /etc/ssl/certs/ca-certificates.crt;
 ```
 
-- [`crowdsec-nginx-bouncer`](https://github.com/crowdsecurity/cs-nginx-bouncer)
+And restart Nginx.
+
+
+### Migrate from v0 to v1
+
+The best way to migrate from the crowdsec-nginx-bouncer v0.* to v1 is to reinstall the bouncer. Indeed, many news configurations options are now available and some has been removed.
+
+- Backup your CrowdSec Local API key from your configuration file (`/etc/crowdsec/bouncers/crowdsec-nginx-bouncer.conf`)
+- Remove the old bouncer:
+
+```bash
+sudo apt-get remove --purge crowdsec-nginx-bouncer
+```
+
+- Install the new bouncer:
 ```bash
-git clone https://github.com/crowdsecurity/cs-nginx-bouncer.git
+sudo apt-get update
+sudo apt-get install crowdsec-nginx-bouncer
 ```
 
-#### Installation
+- Configure the bouncer in `/etc/crowdsec/bouncers/crowdsec-nginx-bouncer.conf`
 
-##### lua-cs-bouncer
 
+## References
+
+### `API_KEY`
 ```bash
-cd ./lua-cs-bouncer/
-sudo make install
+API_KEY=<API_KEY>
 ```
 
-##### crowdsec-nginx-bouncer
+CrowdSec Local API key.
+
+Generated with [`sudo cscli bouncers add`](../cscli/cscli_bouncers_add) command.
 
-- Copy the `crowdsec-nginx-bouncer/nginx/crowdsec_nginx.conf` into `/etc/nginx/conf.d/crowdsec_nginx.conf`:
+### `API_URL`
 ```bash
-cp ./crowdsec-nginx-bouncer/nginx/crowdsec_nginx.conf /etc/nginx/conf.d/crowdsec_nginx.conf
+API_URL=http://<ip>:<port>
 ```
-- Copy the `crowdsec-nginx-bouncer/nginx/access.lua` into `/usr/local/lua/crowdec/access.lua`:
+
+CrowdSec local API URL.
+
+
+### `BOUNCING_ON_TYPE`
 ```bash
-cp ./crowdsec-nginx-bouncer/nginx/access.lua /usr/local/lua/crowdec/access.lua
+BOUNCING_ON_TYPE=all|ban|captcha
 ```
 
-Configure your API url and key in `/etc/crowdsec/bouncers/crowdsec-nginx-bouncer.conf`:
+Type of remediation we want to bounce. 
+If you choose `ban` only and receive a decision with `captcha` as remediation, the bouncer will skip the decision.
 
-```lua
-API_URL=http://127.0.0.1:8080
-API_KEY=<API KEY> --generated with `cscli bouncers add -n <bouncer_name>
-LOG_FILE=/tmp/lua_mod.log
-CACHE_EXPIRATION=1
-CACHE_SIZE=1000
+### `FALLBACK_REMEDIATION`
+```bash
+FALLBACK_REMEDIATION=ban
 ```
 
-You can now restart your nginx server:
+The fallback remediation if the bouncer receive a decision with an unknown remediation.
```

Comment:
The fallback remediation is applied if the bouncer receives a decision with an unknown remediation.